src/reputation/lib.py

```diff
@@ -620,39 +351,80 @@ def __init__(self, withdrawal, balance_record_id, amount):
 
     def complete_token_transfer(self):
         self.withdrawal.set_paid_pending()
-        self.token_payout = self._calculate_tokens_and_update_withdrawal_amount()  # noqa
-        self._request_transfer('RSC')
+        self.token_payout = (
+            self._calculate_tokens_and_update_withdrawal_amount()
+        )  # noqa
+        self._request_transfer("RSC")
 
     def _calculate_tokens_and_update_withdrawal_amount(self):
-        token_payout, blank = ethereum.lib.convert_reputation_amount_to_token_amount(  # noqa: E501
-            'RSC',
-            self.amount
+        (
+            token_payout,
+            blank,
+        ) = ethereum.lib.convert_reputation_amount_to_token_amount(  # noqa: E501
+            "RSC", self.amount
         )
         self.withdrawal.amount = self.amount
         self.withdrawal.save()
         return token_payout
 
     def _request_transfer(self, token):
-        url = ASYNC_SERVICE_HOST + '/ethereum/erc20transfer'
+        url = ASYNC_SERVICE_HOST + "/ethereum/erc20transfer"
         message = {
             "balance_record": self.balance_record_id,
             "withdrawal": self.withdrawal.id,
             "token": token,
             "to": self.withdrawal.to_address,
-            "amount": self.token_payout
+            "amount": self.token_payout,
         }
 
-        contract = w3.eth.contract(abi=contract_abi, address=Web3.toChecksumAddress(RSC_CONTRACT_ADDRESS))
+        contract = w3.eth.contract(
+            abi=contract_abi, address=Web3.toChecksumAddress(RSC_CONTRACT_ADDRESS)
+        )
         amount = int(self.amount)
         paid = False
         to = self.withdrawal.to_address
         tx_hash = execute_erc20_transfer(
-            w3,
-            WEB3_KEYSTORE_ADDRESS,
-            PRIVATE_KEY,
-            contract,
-            to,
-            amount
+            w3, WEB3_KEYSTORE_ADDRESS, PRIVATE_KEY, contract, to, amount
         )
         self.withdrawal.transaction_hash = tx_hash
         self.withdrawal.save()
+
+
+def check_hotwallet():
+    """
+    Alerts admins if the hotwallet is low on eth or RSC
+    """
+
+    contract = w3.eth.contract(
+        abi=contract_abi, address=Web3.toChecksumAddress(RSC_CONTRACT_ADDRESS)
+    )
+    rsc_balance_wei = contract.functions.balanceOf(WEB3_KEYSTORE_ADDRESS).call()
+    decimals = contract.functions.decimals().call()
+    rsc_balance_eth = rsc_balance_wei / (10**decimals)
+    eth_balance_wei = w3.eth.getBalance(WEB3_KEYSTORE_ADDRESS)
+    eth_balanace_eth = eth_balance_wei / (10**18)
+    send_email = False
+    outer_subject = "RSC is running low in the hotwallet"
+
+    if rsc_balance_eth <= 300000:
+        outer_subject = "RSC is running low in the hotwallet"
+        send_email = True
+
+    if eth_balanace_eth < 0.08:
+        outer_subject = "ETH is running low in the hotwallet"
+        send_email = True
+
+    # if send_email:
```

Comment:
Remove

Reply:
woops, this should be in there. it needs to be if send_email is true, then send it..